src/fmeval/eval_algorithms/factual_knowledge.py

```diff
@@ -25,13 +25,55 @@
 from fmeval.transforms.util import validate_call
 from fmeval.util import get_eval_results_path
 
+FACTUAL_KNOWLEDGE = "factual_knowledge"
+FACTUAL_KNOWLEDGE_FUZZY = "factual_knowledge_fuzzy"
+SCORE_NAMES = [FACTUAL_KNOWLEDGE, FACTUAL_KNOWLEDGE_FUZZY]
+
 logger = logging.getLogger(__name__)
 
-FACTUAL_KNOWLEDGE = EvalAlgorithm.FACTUAL_KNOWLEDGE.value
+
+def _exact_inclusion_score(model_output: str, target_output: str) -> float:
```

Comment:
Great name, I think we should perhaps use those names as the score names directly (see general comment on naming).